R/utils.R

```diff
@@ -207,6 +209,18 @@ parse_database_error <- function(msg) {
   )
 }
 
+contextualize_database_error <- function(cnd_body) {
```

Comment:
Extracted this out as a function in case we're interested in applying this to other common messages that we can't resolves ourselves behind-the-scenes.

Reply:
https://github.com/r-dbi/odbc/issues/653 is another situation where I imagine us doing this. With an expired token, the databricks message essentially reads like a 403, but all the user actually has to do is refresh their token.